lib/buildingsync/model_articulation/building.rb

```diff
@@ -266,7 +266,7 @@ def set_weater_and_climate_zone(weather_file_name, climate_zone)
         weather_file = "CZ01RV2.epw"
       elsif climate_zone == "2A" || climate_zone == "2B"
         weather_file = "CZ02RV2.epw"
-      elsif climate_zone == "3A" || climate_zone == "3B" || climate_zone == "3C"
+      elsif climate_zone == "3A" || climate_zone == "3B" || climate_zone == "3C" || climate_zone == "Climate Zone 3"
```

Comment:
I think this should be coming from OS Standards somehow

Reply:
This needs to be discussed. We added support to provide an EPW file directly. I have not seen a function like this in Standards.

Reply:
see issue #14